src/components/Callout.astro

```diff
@@ -0,0 +1,50 @@
+---
+import callout_889 from '../assets/images/callout_889.svg'
+import callout_map from '../assets/images/callout_map.svg'
+import callout_forum from '../assets/images/callout_forum.svg'
+---
+<section class="sp_callout">
+    <div class="grid-container margin-y-6">
+        <div class="grid-row grid-gap ">
+          <div class="tablet:grid-col-6">
+            <img
+             src ={callout_889}
+             alt ="Three panels from a screen, the first panel has a magnifying glass and reads use the 889 search to find your vendor, the second panel has a green check mark and reads in compliance with Section 889, and third panel has a red exclamation mark and reads not in compliance with Section 889."
+            >
+          </div>
+          <div class="tablet:grid-col-6">
+            <h2>Section 889 Search</h2>
+            <p>Our Section 889 search tool makes it easy to check an entity’s Section 889 status in <a href="https://sam.gov/">SAM.gov</a>.</p>
```

Comment:
I wonder if we need the external link icon on this. It's not in the mockup, but I think it's required.